README.md

```diff
@@ -1 +1,20 @@
-# nlp-servic
\ No newline at end of file
+# NLP-Service for entity extraction using spacy or Polyglot depending on the language
+The Service provides entities and detects the language of text. Results are served by a flask app with two endpoints:
+`entity_extraction`, which returns the entities in the text, the used method/model and the language of the text. You can also add a language in the POST-request, in order to specify, which language should be used by the service.
```

Comment:
format this as a markdown list so it displays properly. Also, show available routes like `POST /entity_extraction`